cli/pkg/chart/chart.go

```
@@ -111,6 +111,11 @@ func CollectChartInstallValues(values *tt.TonicChartValues, r io.Reader, w io.Wr
 			chartValues.TonicAI.OracleHelperConfig = nil
 		})
 
+	err := getLatestTonicVersion(collector);
+	if err != nil {
+		return nil, err;
+	}
+
```

We'll be better off doing this in `CollectBaseTonicValues` instead of here because we can just set version tag directly on the configuration object:

```go
func CollectBaseTonicValues(...) ... {
    if values == nil { .. }
    values.Version = getLatestTonicVersion()
    collector := ...
}
```

And instead of `getLatestTonicVersion` returning `err` it can always return a string because even if it failed to get the actual latest version it can just return `latest`

---

request granted